extension/src/openvic-extension/singletons/CursorSingleton.cpp

```diff
@@ -0,0 +1,586 @@
+#include "CursorSingleton.hpp"
+
+#include <vector>
+
+#include <godot_cpp/classes/dir_access.hpp>
+#include <godot_cpp/classes/input.hpp>
+#include <godot_cpp/classes/scene_tree.hpp>
+#include <godot_cpp/core/error_macros.hpp>
+#include <godot_cpp/variant/string.hpp>
+
+#include <openvic-extension/utility/ClassBindings.hpp>
+#include <openvic-extension/utility/Utilities.hpp>
+
+using OpenVic::Utilities::godot_to_std_string;
+using OpenVic::Utilities::std_to_godot_string;
+
+using namespace godot;
+using namespace OpenVic;
+using namespace OpenVic::NodeTools;
```

Comment:
Is this needed in this file?

Reply:
We do use godot for various datatypes. Namespace OpenVic is used for GameSingleton access only to my knowledge.
Removed NodeTools because it is completely unused.